moj-node/test/routes/content.spec.js

```diff
@@ -187,17 +204,29 @@ describe('GET /content/:id', () => {
           const $ = cheerio.load(response.text);
 
           expect($('#title').text()).to.include(
-            'Foo article',
+            'The jobs noticeboard',
             'Page title did not match',
           );
           expect($('#stand-first').text()).to.include(
-            'Foo article stand first',
+            'Some incredible foo stand first',
             'Article stand first did not match',
           );
           expect($('#body').text()).to.include(
-            'Foo article body',
+            'Foo paragraph',
             'Article body did not match',
           );
+
+          // suggestions
+          expect($('#suggested-content a').length).to.equal(1);
```

Comment:
Nice.. I know this is tech debt but any chance you can break up this test..
As in one render and 3 smaller tests?

Reply:
@romidane done I believe!